src/sentry/workflow_engine/endpoints/validators/base/action.py

```diff
@@ -60,6 +61,22 @@ def _check_action_type(self, action_type: Action.Type) -> None:
                 f"Organization does not allow this action type: {action_type}"
             )
 
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        from sentry.notifications.notification_action.registry import action_validator_registry
+
+        attrs = super().validate(attrs)
+
+        if not (organization := self.context.get("organization")):
+            raise serializers.ValidationError("Organization is required in the context")
+
+        try:
+            handler = action_validator_registry.get(attrs["type"])
+        except NoRegistrationExistsError:
+            # TODO: remove try/catch when all existing action types are registered
```

Comment:
we'll probably want to leave this as the default case, so we can introduce new action types easily in the future.